assets/src/design-system/theme/helpers/scrollbar.js

```diff
@@ -32,6 +32,7 @@ export const scrollbarCSS = () => css`
 
   ::-webkit-scrollbar {
     width: 11px;
```

Comment:
Should these use the scrollbar width constants?

Reply:
Then it should be moved to the theme, but yeah, makes sense.

Reply:
Moved in 60b9275e2.